p-token/src/processor/initialize_mint.rs

```diff
@@ -71,6 +75,8 @@ pub fn process_initialize_mint(
 }
 
 /// Instruction data for the `InitializeMint` instruction.
+/// JC: for legibility and safety, can this hold onto the ref instead of a pointer?
+/// Or does that introduce too many bounds checks and increase CUs a lot?
```

Comment:
It will use more CUs, but probably not a lot. I will test it.

Reply:
Working only with the slice increases CU by `12`, which is about `10%` of the CU of the instruction.